lib/render_template.rb

```diff
@@ -1,6 +1,10 @@
+require 'yaml'
+
 class RenderTemplate
   def self.render_template(configuration, environment, config, version)
```

Comment:
While we're editing this function, I wonder if it's worth giving these parameters more specific names than `configuration` and `config`.